runtime/battery-station/Cargo.toml

```diff
@@ -148,7 +148,7 @@ parachain = [
 
     # XCM
 
-    "kusama-runtime",
+    "rococo-runtime",
```

Comment:
These seem to be ordered lexicographically, but `"rococo-runtime"` is not sorted in correctly.

Reply:
67906fc019237c8c122e016c8d4f02d0052b4926